src/ensembl/src/content/app/browser/Browser.scss

```diff
@@ -10,7 +10,7 @@
   margin-top: 50px;
   padding-left: 40px;
 
-  &__emptyTopBar {
+  &__emptyTopbar {
```

Comment:
out of curiosity, why do you prefer Topbar to TopBar? Sidebar has merged to form a separate word, but I have never seen top bar do the same.

Reply:
I saw it in a few places when I did a quick Google search. And then there was this for the inspiration: https://trigger.io/modules/topbar/current/docs/index.html